reconcile/test/test_quay_repos_api.py

```
@@ -0,0 +1,118 @@
+
+    with pytest.raises(IntegrationError, match="duplicate repo name"):
+        integration.compile_desired_state(orgs=[org], apps=[app1, app2])
+
+
+def test_compile_desired_state_duplicate_includes_org_context() -> None:
+    integration = _make_integration()
+    org = _make_org(name="myorg", instance_name="quay.io")
+    app1 = _make_app("myorg", "quay.io", ["conflict"])
+    app2 = _make_app("myorg", "quay.io", ["conflict"])
+
+    with pytest.raises(IntegrationError, match="quay.io/myorg"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)